### DISCUSSION TITLE
我发现一个严重bug （ I found a serious bug）

### DISCUSSION THREAD

**Discussion by fengmai** [air-verse/air#386] [Ideas]
example：

 aa:="4"
 aa="%"+aa
fmt.Println(aa)

 正确获取应该是  %4   但是确得到是%！
The correct answer should be   %4 ,but is %！

**Comment by makiuchi-d** [comment]
```diff
- fmt.Println(a)
+ fmt.Println(aa)
```
serious bugs are always in your code.

**Reply by fengmai** [reply]
我这边是少打一个a，你可以用fresh编译后看一下这个结果
I( want to type a less. You can use fresh to compile and see the result）
